src/main/java/de/tum/in/www1/artemis/web/rest/TextAssessmentResource.java

```diff
@@ -321,23 +317,17 @@ private void computeBlocks(Result result, TextExercise textExercise) {
      * @param submissionId the id of the submission
      * @return the result linked to the submission
      */
-    @GetMapping("/exercise/{exerciseId}/submission/{submissionId}/exampleAssessment")
+    @GetMapping("/exercise/{exerciseId}/submission/{submissionId}/example-result")
     @PreAuthorize("hasAnyRole('TA', 'INSTRUCTOR', 'ADMIN')")
-    public ResponseEntity<Result> getExampleAssessmentForTutor(@PathVariable Long exerciseId, @PathVariable Long submissionId) {
+    public ResponseEntity<Result> getExampleResultForTutor(@PathVariable Long exerciseId, @PathVariable Long submissionId) {
```

Comment:
```suggestion
    public ResponseEntity<Result> getExampleResultForTutor(@PathVariable long exerciseId, @PathVariable long submissionId) {
```
hope I learned my lesson ;)